numpy/core/tests/test_multiarray.py

```diff
@@ -2497,6 +2497,15 @@ def test_diagonal_memleak(self):
         if HAS_REFCOUNT:
             assert_(sys.getrefcount(a) < 50)
 
+    def test_size_zero_memleak(self):
+        # Regression test for issue 9615
+        a = np.array([])
```

Comment:
I did like the dtype last time, as it makes clear we are testing floats, so a simple
`np.array([], dtype=np.float64)` would be good.

Reply:
Might add a comment explaining the need for float because the problem was in `cblasfuncs.c`.